code/modules/power/tracker.dm

```diff
@@ -23,11 +23,11 @@
 
 /obj/machinery/power/tracker/Destroy()
 	unset_control() //remove from control computer
-	return ..()
+	..()

```

Comment:
Destroy() procs should always return the parent value, either by `return ..()` or `. = ..()`